hw_2/tasks.py

```diff
@@ -0,0 +1,41 @@
+import random
+import string
+
+
+# Write a code, which will:
+# 1. create a list of random number of dicts (from 2 to 10)
+# dict's random numbers of keys should be letter,
+# dict's values should be a number (0-100),
+# example: [{'a': 5, 'b': 7, 'g': 11}, {'a': 3, 'c': 35, 'g': 42}]
+
+# 2. get previously generated list of dicts and create one common dict:
+# if dicts have same key, we will take max value, and rename key with dict number with max value
+# if key is only in one dict - take it as is,
+# example: {'a_1': 5, 'b': 7, 'c': 35, 'g_2': 42}
+
+number_of_dictionaries = random.randint(2, 10)
+list_of_dictionaries = []
+
+for dictionary in range(0, number_of_dictionaries):
+    c = {}
```

Comment:
Better use naming for ex `temp_dict`, so it's clearer that variable stores temporary dictionary